adminSiteClient/GrapherConfigGridEditorTypesAndUtils.tsx

```diff
@@ -167,6 +173,7 @@ export interface ColumnDataSourceUnknown {
 }
 export type ColumnDataSource =
     | ColumnDataSourceFieldDescription
+    // | ColumnDataSourceMultipleFieldDescriptions
```

Comment:
I'm not sure if this should really be commented out?

Reply:
Deleted it - I thought I might need to have editors be able to edit several properties at once but for now we can make it work with just a single property (as colorScale is a single property that is a JS object but for mapping to columns it's a single entity so the mapping to multiple entities is not needed for now)